PocketKit/Sources/PocketKit/Settings/DeleteAccountView.swift

```diff
@@ -0,0 +1,143 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Textile
+
+@MainActor
+struct DeleteAccountView: View {
+    /// If the user is premium
+    @State var isPremium: Bool
+
+    /// Confirmation by the user that they have cancelled their premium account
+    @State var hasCancelledPremium: Bool = false
+
+    /// Confirmation by the user they they understand the deletion is permanent
+    @State var understandsPermanentDeletion: Bool = false
+
+    /// State variable listened on from our view model
+    @State var isPresentingCancelationHelp: Bool
+
+    var deleteAccount: () -> Void
+
+    @Environment(\.dismiss)
+    var dismiss
+
+    var body: some View {
+        NavigationView {
+            VStack(spacing: 0) {
+                Text(L10n.Settings.AccountManagement.deleteYourAccount)
+                    .style(.deleteAccountView.header)
+                    .padding(.top, 50)
+                    .padding()
+
+                Text(L10n.Settings.AccountManagement.DeleteAccount.warning)
+                    .style(.deleteAccountView.warning)
+                    .frame(maxWidth: .infinity, alignment: .leading)
+                    .padding()
+
+                /**
+                 Note below we use LocalizedStringKey. This is a total hack. This is because a SwiftUI text object treats Text("testing") differently then Text(someText). When passed as a variable, the Text view performs no localization, but it also does not render the embedded markdown. When cast to a LocalizedStringKey the Text field will try and localize the content (and fail) but it will render the markdown and bold the text.
```

Comment:
This was fun.. thanks @JohnOberhauser for the 🦆

Reply:
@cyndichin nope, its cause of how SwiftUI worked. Remind me to show you in office hours. its the most annoying thing.